alacritty_terminal/src/tty/unix.rs

```diff
@@ -232,7 +242,11 @@ pub fn new(config: &PtyConfig, window_size: WindowSize, window_id: Option<usize>
         },
         Err(err) => Err(Error::new(
             ErrorKind::NotFound,
-            format!("Failed to spawn command '{}': {}", shell.program(), err),
+            format!(
+                "Failed to spawn command '{}': {}",
```

Comment:
Wondering if we should just log the full command here.

Reply:
We shouldn't.